bin/p2-replicate/main.go

```diff
@@ -77,6 +87,27 @@ func main() {
 	replicator.Logger.Logger.Formatter = new(logrus.TextFormatter)
 	replicator.MinimumNodes = *minNodes
 
+	if *skip {
+		status, err := healthChecker.LookupHealth(manifest.ID())
+		if err != nil {
+			replicator.Logger.WithField("err", err).Errorln("Could not retrieve health checks to resolve skipped nodes")
+			os.Exit(1)
+		}
+
+		var newHosts []string
+		for _, host := range *hosts {
+			thisHealth, _ := status.ForNode(host)
+			if thisHealth.Health != health.Passing && thisHealth.Health != health.Unknown {
```

Comment:
Shouldn't this if statement be skipping the `append` if it evaluates to true? Also, shouldn't the levels it checks against be the one specified by the `ServiceUpgrader`?